operating_unit/models/res_users.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent - Jordi Ballester Alomar
+# © 2015 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+
+    _inherit = 'res.users'
+
+    @api.model
+    def operating_unit_default_get(self, uid2):
+        if not uid2:
+            uid2 = self._uid
+        user = self.env['res.users'].browse(uid2)
+        return user.default_operating_unit_id and \
+            user.default_operating_unit_id.id
+
+    @api.model
+    def _get_operating_unit(self):
+        return self.operating_unit_default_get(self._uid)
+
+    @api.model
+    def _get_operating_units(self):
+        op_unit = self.operating_unit_default_get(self._uid)
+        if op_unit:
+            return [op_unit]
+        return False
```

Comment:
If using recordsets instead of IDs, probably this can be just `return self._get_operating_unit()`.
It's good to still have two methods, for better extensibility.